CMakeLists.txt

```diff
@@ -42,6 +42,8 @@ set(TLRENDER_QT5 FALSE CACHE BOOL "Enable support for Qt5")
 set(TLRENDER_PROGRAMS TRUE CACHE BOOL "Build programs")
 set(TLRENDER_EXAMPLES TRUE CACHE BOOL "Build examples")
 set(TLRENDER_TESTS TRUE CACHE BOOL "Build tests")
+set(TLRENDER_GCOV FALSE CACHE BOOL "Enable gcov code coverage")
```

Comment:
Could you check that you are synced with that latest? These coverage changes don't seem to be related to the PR.

Reply:
Fixed.